### PR TITLE
docs(goctl): goctl 1.3.4 migration note

### PR DESCRIPTION
**Pull request by fynxiu**
* adds a simple lang check
* adds migration notes

### CODE REVIEW

**Review by kevwan — APPROVED** [review]
lgtm.

I think it can be removed after couple releases.

**Comment by fynxiu** [comment]
> lgtm.
> 
> I think it can be removed after couple releases.

I think so too.

File: tools/goctl/model/sql/command/migrationnotes/v1.3.4.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/urfave/cli"
+)
+
+func migrateBefore1_3_4(ctx *cli.Context) error {
+	// TODO: Hard-coded arguments should be eliminated
```
Comment (anqiansong) [inline tools/goctl/model/sql/command/migrationnotes/v1.3.4.go:12]: Do not remain TODO in code

**Review by anqiansong — APPROVED** [review]

**Review by kevwan — APPROVED** [review]
lgtm.

**Pull request merged**